lib/config/recommended.js

```diff
@@ -16,4 +17,14 @@ module.exports = {
   plugins: ['ember'],
 
   rules,
+
+  overrides: [
+    {
```

Comment:
Can you add a high-level comment (maybe just a sentence) explaining what this override is for?

Reply:
yeah, generally, I think the whole eslint ecosystem should move away from top level configs use overrides-only based approaches. A lot of the complexity from eslint configs comes from cascading configs and different directory depths, and using overrides specifically is a lot easier to understand.

But _that_ is a topic for another time. :upside_down_face: 

I'll add something specifically fro gjs/gts. thanks!

Reply:
done